Makefile

```diff
@@ -25,7 +25,15 @@ ${FISSILE_BINARY}: bin/dev/install_tools.sh
 ########## VAGRANT VM TARGETS ##########
 
 certs: uaa-certs
-	${GIT_ROOT}/bin/generate-dev-certs.sh cf bin/settings/certs.env
+	# If DOMAIN is set, place DOMAIN and UAA_HOST at the end of network.env temporarily
+	# Line continuations are necessary to trap EXIT after running script generation
+	if test -n "${DOMAIN}"; then \
+		echo "DOMAIN=${DOMAIN}" >> bin/settings/network.env; \
+		echo "UAA_HOST=uaa.${DOMAIN}" >> bin/settings/network.env; \
+		trap "sed -i '$d' bin/settings/network.env && \
+			sed -i '$d' bin/settings/network.env" EXIT; \
```

Comment:
I was able to find the right combination of escaping for both Make and shell to get this working... see updates! Thanks for finding this